ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Start script for promise-to-file-web
+
+PORT=3000
+
+export NODE_PORT=${PORT}
+exec node /opt/dist/App.js -- ${PORT}
```

Comment:
This should be www.js, similar to here: https://github.com/companieshouse/confirmation-statement-web/blob/main/ecs-image-build/docker_start.sh